### ISSUE TITLE
This is a Trojan!

### ISSUE THREAD

**Issue by Wild-Alex** [macobra52/SMB2-Randomizer#1]
Downloaded the randomizer, but my antivirus detected it as a threat, I used VirusTotal to check the issue, and there's a trojan in this, it is not safe to download!

And if this isn't intended, devs, fix it please

![Screenshot 2024-01-21 044759](https://github.com/macobra52/SMB2-Randomizer/assets/108005472/9a088adf-428a-43ae-b677-8145482b34db)

**Comment by macobra52** [comment]
I assure you, any detected malware in the randomizer is a false positive (seems the two vendors reporting it are known for false positives, doing a quick Google search)

Basically, I use a utility called [Launch4j](https://launch4j.sourceforge.net/) to create an exe to assist with launching the Java executable (JAR). This is technically not needed, but since people are more familiar with exe files and it does help by presenting a message if Java happens to not be installed/installed improperly, I decided to incorporate it. All the exe does it attempt to launch SMB2Randomizer.jar in the same directory, using the recommended settings to minimize false positives.

In short, if you're still uncomfortable running the exe launcher, you can simply delete it and run SMB2Randomizer.jar instead and it should work (if Java 8+ is installed properly). Again though, there are no trojans or any malware contained here. Those are false positives
